docs/Manufacturer Design Guidelines.md

```diff
@@ -0,0 +1,357 @@
+# Betaflight Flight Controller Manufacturer Design Guidelines
+
+
+![Betaflight](docs/assets/images/bf_logo.png)
+
+
+Version Change Register
+
+| Version # | Revision Date | Changes, Reasons, and Notes |
+| :--- | :--- | :--- | 
+| Draft 0.1 | 14 May 2022 | Initial Draft Format |
+| Draft 0.2 | 04 June 2022 | Revise format to Final Format |
+| Draft 0.3 | 12 June 2022 | Update Visual Media and Tables |
+| Draft 0.4 | 21 October 2022 | Update format, add information |
+| Draft 0.5 | 24 October 2022 | Add additional information |
+
+
+Thank you for considering or continuing your development of Betaflight capable flight control hardware.  
+
+Betaflight is an open source project that is free to use and does not incur a license cost, however for the most successful release of a new flight controller or complete ready-to-fly product that is using Betaflight, it remains immensely beneficial to provide representative production samples or pre-production testing units to the Betaflight development team for testing and development feedback.  
+
+In order to have hardware added to the Betaflight approved hardware list, hardware samples representative of the final configuration must be provided to designated members of the betaflight development team.  It is strongly recommended that samples of development hardware or production-representative examples are evaluated positively before accepting pre-orders or releasing products.  Many of the same benefits can be provided for inclusive hardware packages, including flight control stacks, or ready-to-fly craft.
+
+Sharing schematics/layouts with the reviewers will also be beneficial and improve the quality of the review.
+
+This provides the Betaflight team with an opportunity to ensure that the hardware and firmware behave 100% as expected in the representative configuration, as well as support verification of custom defaults required for firmware operation.  After-sales support to customers from members of the Betaflight team are also made possible for reproducing end user issues.
+
+Additional benefits are also present in the form of allowing experienced active pilots with backgrounds in engineering of these systems to assist with aspects of the development process particularly in respect to real world use the products will be subjected to. 
+
+Finally, we will offer a ‘developer approved’ product list on the Betaflight github to advise the userbase on electronics which both follow our ‘best practice’ guidance, and which have been tested by our development team. This will be available for flight controller hardware, as well as electronic speed controller stacks, AIOs, and ready-to-fly craft. This strategy is designed to help both developers optimise their hardware and our user base get directed to optimal hardware, and reduce support requests for the Betaflight team who serve a user base of over a hundred thousand users.
```

Comment:
can we change this to "Finally, we will offer a ‘Betaflight approved’ product list on the Betaflight github...'developer' sounds too vague

Reply:
Fixed